### PR TITLE
Add v2 updates api

### PR DESCRIPTION
**Pull request by eherget**
This adds a v2 updates api, which no longer returns package summary and description.

### CODE REVIEW

File: webapp/app.py [hunk 1]
```diff
@@ -849,7 +977,8 @@ def setup_apispec(handlers):
     })
     # Register public API handlers to apispec
     for handler in handlers:
-        if handler[0].startswith(r"/api/v1/"):
+        if handler[0].startswith(r"/api/v1/") \
```
Comment (ggainey) [inline webapp/app.py]: We could write this with r"/api/v[12]/" instead of two compares, yeah?

Comment (eherget) [reply webapp/app.py]: I tried doing that, but it didn't work.  Spent an hour trying to figure out what the correct regex is.  Gave up and did this.  :(

**Comment by eherget** [comment]
If no Jenkins tests failed, why is it set as failing?

**Comment by eherget** [comment]
This fixed Issue #266

**Pull request merged**